packages/components/addon/components/hds/tabs/tab-panel.hbs

```diff
@@ -0,0 +1,11 @@
+{{! Hds-TabPanel }}
+<section
+  class="hds-tab-panel"
+  role="tabpanel"
+  {{!-- id="section1" --}}
+  {{!-- aria-labelledby="tab1" --}}
+  {{! hidden TODO: set by arg }}
+  ...attributes
```

Comment:
Added this temporarily. Don't know if it will be needed or even desirable for final implementation.

Reply:
Very likely we will keep it. We tend to allow consumers/users to pass attributes to the components (eg to add 'data-test' attributes or extra class names).